dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerTest.java

```diff
@@ -219,7 +222,7 @@ protected String parseResponse(WebResponse wr, String beginText, String endText)
      *                       if something goes horribly wrong
      */
     @Test
-    @SkipForRepeat("SERVLET-4.0")
+    @SkipForRepeat({"SERVLET-4.0", EE9_FEATURES})
```

Comment:
We're just skipping here because this is a silly test to repeat three times I assume?